tests/api/PDFDocument.spec.ts

```diff
@@ -123,6 +124,43 @@ describe(`PDFDocument`, () => {
     });
   });
 
+  describe(`embedFont() method`, () => {
```

Comment:
Can we set `updateMetadata` to `false` in both of these tests? I'm worried about the potential for flakiness if we don't.